sdk/tables/aztable/options.go

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
As a code hygiene issue, it is better to not have an options.go file at all. It is better to put the option structs near the methods that use them. The reason is because a change to the option struct means there is a change to the method and so they should be near each other.